generators/client/templates/angular/src/main/webapp/app/admin/configuration/_configuration.component.html

```diff
@@ -3,12 +3,12 @@ <h2 jhi-translate="configuration.title">Configuration</h2>
 
     <span jhi-translate="configuration.filter">Filter (by prefix)</span> <input type="text" [(ngModel)]="filter" class="form-control">
     <label>Spring configuration</label>
-    <table class="table table-condensed table-striped table-bordered table-responsive"
+    <table class="table table-sm table-striped table-bordered"
```

Comment:
It seems so Sendil, thanks ! But it's quite interesting, in some pages I have to wrap the element or it doesn't work as expected ( if you have the time try it in audits.component.html)